### PR TITLE
Add info/link about Animate.css to README

### PR DESCRIPTION
**Pull request by floer32**
For newbs to Animate CSS (like me). It took me a minute to realize that it was a separate library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,15 @@ Clone this repository, then run:
...
 ```
 
-## Deploy to Github Pages
+## Notes
+
+### [Animate CSS][1]
+
+* This demo uses additional CSS classes for animations, like `fadeIn`, which are from [`tailwindcss-animatecss`](https://github.com/bentzibentz/tailwindcss-animate.css), a [Tailwind CSS plugin](https://tailwindcss.com/docs/plugins) that integrates [Animate CSS][1].
+
+[1]: https://github.com/animate-css/animate.css)
```
Comment (michelegera) [inline README.md]: Please remove the extra `)` at the end of the link.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,15 @@ Clone this repository, then run:
 $ yarn dev
 ```
 
-## Deploy to Github Pages
+## Notes
+
+### [Animate CSS][1]
```
Comment (michelegera) [inline README.md]: Would you mind changing the spelling to `Animate.css` both here and below?

**Review by michelegera — CHANGES_REQUESTED** [review]

**Comment by michelegera** [comment]
@hangtwenty thanks for your contribution!

I think this needs a couple trivial fixes and then it’s ready to merge 😄

File: README.md (hunk 1 above)
Comment (floer32) [reply README.md]: 🤦  Thanks for catching that

**Review by michelegera — APPROVED** [review]

**Pull request merged**